src/transform.py

```diff
@@ -0,0 +1,48 @@
+from datetime import datetime
+import pandas as pd
+
+
+def read(file_path:str)-> pd.DataFrame:    
+    return pd.read_csv(file_path)
+
+df1 = read('data/sales.csv')
+df2 = read('data/sensor_stock_levels.csv')
+df3 = read('data/sensor_storage_temperature.csv')
+
+# Convert 'timestamp' column to datetime
+df1['timestamp'] = pd.to_datetime(df1['timestamp'], format='%Y-%m-%d %H:%M:%S')
```

Comment:
you can add this code to below function so you won't need to repeat this 3 times.